webpack/webpack.prod.config.js

```diff
@@ -170,5 +171,11 @@ module.exports = Merge.smart(commonConfig, {
       license: process.env.NEW_RELIC_LICENSE_KEY || 'fake_app',
       applicationID: process.env.NEW_RELIC_APP_ID || 'fake_license',
     }),
+    new NewRelicSourceMapPlugin({
+      applicationId: process.env.NEW_RELIC_APP_ID,
+      nrAdminKey: process.env.NEW_RELIC_ADMIN_KEY,
```

Comment:
Is this a thing that is provided by GOCD?

Reply:
I'm not sure if `NEW_RELIC_ADMIN_KEY` is the thing you are referring to?  I do need to tie this in to GoCD if this works.  I have created a DEVOPS ticket (DEVOPS‌-8600) to help me verify as I continue to finish out that work.